recipes/configure_server.rb

```diff
@@ -24,8 +24,13 @@
   end
 end
 
-# install chef-vault if needed
-include_recipe 'chef-vault' if node['percona']['use_chef_vault']
+if node['percona']['use_chef_vault']
+  chef_gem 'chef-vault' do
```

Comment:
If you want to require chef 13.4+ then you can remove this altogether since we shipped chef-vault in chef-client.